cmd/client/main.go

```diff
@@ -126,12 +127,33 @@ func (c *Client) createSAToken(ctx context.Context, audience string, sa, namespa
 
 }
 
-func (c *Client) initGRPCClient(cacert []byte, URL string) {
+func (c *Client) streamClientInterceptor(token, namespace string) grpc.StreamClientInterceptor {
+	return func(
+		ctx context.Context,
+		desc *grpc.StreamDesc,
+		cc *grpc.ClientConn,
+		method string,
+		streamer grpc.Streamer,
+		opts ...grpc.CallOption,
+	) (grpc.ClientStream, error) {
+		return streamer(c.attachToken(ctx, token, namespace), desc, cc, method, opts...)
+	}
+}
+
+func (c *Client) attachToken(ctx context.Context, token, namespace string) context.Context {
+	return metadata.AppendToOutgoingContext(ctx, "authorization", token, "namespace", namespace)
```

Comment:
Shouldn't pass namespace via the metadata.

Reply:
I have remove this